core/arch/arm/include/mm/core_mmu.h

```diff
@@ -396,4 +404,7 @@ typedef enum {
 /* Check cpu mmu enabled or not */
 bool cpu_mmu_enabled(void);
 
+/* alloc and fill SDP memory objects table - table is NULL terminated */
```

Comment:
Nitpicking - please start comments with an uppercase letter (applies to most of your comments)